### ISSUE TITLE
Feature Request: delete articles manually or in settings, set x days to automatically delete articles

### ISSUE THREAD

**Issue by kNsja5w** [Ranchero-Software/NetNewsWire#4163]
Right now, articles can be set to read and unread only. If you wish to hide read articles (or articles you will never read, because they're unimportant), you have to mark and filter them. Otherwise there's no chance to get rid of them. I think the overall UX could be improved by 

1. including a manual **delete button** and / or 
2. by adding a feature into settings, which automatically deletes articles after x days (the days can be set by the user)

This would help to reduce the need to mark "unread" articles to "read" to hide them and also to get rid of the count of unread articles next to the feed itself.

<img width="223" alt="Bildschirmfoto 2023-12-02 um 12 42 50" src="https://github.com/Ranchero-Software/NetNewsWire/assets/16246095/b14060f6-a5dd-4066-9fe5-ea0c9a614207">

**Comment by tyeNewton** [comment]
I agree with this and, while individual articles aren’t large, I also think it’s important in order to clean up storage consumption.

**Comment by brentsimmons** [comment]
Articles do get marked as read automatically, and then they’re deleted later. They won’t just pile up forever. (How long before that happens is different with each syncing system.)

Also note that you can Mark All as Read — and even Mark Above as Read and Mark Below as Read — when you want to mark a whole bunch of articles as read at once.
